src/Horde.jl

```diff
@@ -1,5 +1,5 @@
 module Horde
 
-greet() = print("Hello World!")
+include("GVF/GVF.jl")
```

Comment:
Does this automatically re-export things in the GVF.jl file? For instance, would we have to do:
```julia
include("GVF/GVF.jl")
export GVF
```
to access the `GVF` struct outside of the Horde module?

Reply:
Hmm yeah that could be cleaner. exporting in the respective files was done because they were originally their own modules. I'll update that

Reply:
The include basically copy pastes code like in c++, so export is carried over from the file.